lib/installed-packages-panel.coffee

```diff
@@ -1,6 +1,6 @@
 _ = require 'underscore-plus'
 {$$, TextEditorView, View} = require 'atom-space-pen-views'
-{Subscriber} = require 'emissary'
+{CompositeDisposable} = require 'event-kit'
```

Comment:
Sure!

Just followed the libraries' README example. May we need to add a comment
about this in there?

I'll update this later - thanks!